libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/Text.kt

```diff
@@ -56,13 +56,18 @@ fun Text(
     text: String,
     modifier: Modifier = Modifier,
     color: Color = Color.Unspecified,
+    // Will be removed, only style should be used
```

Comment:
Are we sure we want to remove this customization? I agree we shouldn't use them most of the time, but there might be some cases where we might need to override them.

Reply:
If this is really necessary to override, we could still write

```kotlin
style = ElementTheme.typography.fontBodyMdMedium.copy(fontSize = 25.sp)
```